internal/cfgplugins/policyforwarding.go

```diff
@@ -263,7 +281,13 @@ func InterfacePolicyForwardingConfig(t *testing.T, dut *ondatra.DUTDevice, a *at
 		switch dut.Vendor() {
 		case ondatra.ARISTA: // Currently supports Arista devices for CLI deviations.
 			// Format and apply the CLI command for traffic policy input.
-			helpers.GnmiCLIConfig(t, dut, fmt.Sprintf("interface %s.%d \n traffic-policy input tp_cloud_id_3_%d \n", aggID, a.Subinterface, a.Subinterface))
+			cliConfig := ""
```

Comment:
Use `var` style syntax for zero/default value declarations.

https://google.github.io/styleguide/go/best-practices#declaring-variables-with-zero-values
```suggestion
			var cliConfig string
```